wimm.Secundatives/Range.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+
+namespace wimm.Secundatives
+{
+    /// <summary>
+    /// A range for an <see cref="IComparable{T}"/> value.
+    /// </summary>
+    public class Range<T> where T : IComparable<T>
+    {
+        /// <summary>
+        /// The minimum value.
+        /// </summary>
+        public T Min { get; }
+
+        /// <summary>
+        /// The maximum value.
+        /// </summary>
+        public T Max { get; }
+
+        /// <summary>
+        /// Indicates whether <see cref="Min"/> and <see cref="Max"/> are excluded from the range.
+        /// When <c>false</c>, values be greater than or equal to <see cref="Min"/> and less than
+        /// or equal to <see cref="Max"/> are considered in range. When <c>true</c> values must be
+        /// strictly greater than <see cref="Min"/> and less than <see cref="Max"/>.
+        /// </summary>
+        public bool Exclusive { get; }
```

Comment:
I'm not really 100% on this generally given that most the time ranges are lower bound inclusive upper bound exclusive.

Reply:
Also not 100% on the paradigm of this as a member/includes. I think just presenting and constraining min/max and throwing probably meets most needs without adding complexity

Reply:
I'm fine with removing the support for `Exclusive` , though I would prefer both bounds inclusive over lower bound inclusive upper bound exclusive. I think I think the common behavior is start/count rather than inclusive-min/exclusive-max. ICBC

Reply:
I'm not too picky. Pretty sure python does:
range(start, end, increment) -- Half open
rust does something similar & half open as well. 


Realistically I think that the base structure of from:to is a good start

Reply:
resolved?